graviti/manager/sheets.py

```diff
@@ -55,13 +57,18 @@ def _init_data(self) -> None:
             sheet = self._get_sheet(sheet_name)
             schema = PortexType.from_yaml(sheet["schema"])
 
+            patype = schema.to_pyarrow()
+
             factory = LazyFactory(
                 sheet["record_count"],
                 128,
                 partial(self._list_data, sheet_name=sheet_name),
-                schema.to_pyarrow(),
+                pa.struct([pa.field(RECORD_KEY, pa.string()), *patype]),
             )
+
             paging_lists = factory.create_lists(schema.get_keys())
+            paging_lists[RECORD_KEY] = factory.create_list((RECORD_KEY,))
```

Comment:
Why not add `(RECORD_KEY,)` to the keys and save the `create_list` call?

Reply:
`create_lists` also calls `create_list`, is there any difference?